uvicorn/importer.py

```diff
@@ -1,11 +1,13 @@
 import importlib
+from types import ModuleType
+from typing import Union
 
 
 class ImportFromStringError(Exception):
     pass
 
 
-def import_from_string(import_str):
+def import_from_string(import_str: Union[ModuleType, str]) -> ModuleType:
```

Comment:
@Kludex @euri10 This has raised `'ModuleType' object is not callable` warnings all over (in my IDE - PyCharm).
For example:
https://github.com/encode/uvicorn/blob/bf1c64e2c141971c546671c7dc91b8ccf0afeb7d/uvicorn/config.py#L314
https://github.com/encode/uvicorn/blob/bf1c64e2c141971c546671c7dc91b8ccf0afeb7d/uvicorn/config.py#L356
- Is this a valid issue?
- Should we switch to `Any` as return type annotation?